packages/axios-extension/src/auth/uaa.ts

```diff
@@ -86,6 +86,24 @@ export class Uaa {
         return this.serviceInfo.systemid;
     }
 
+    /**
+     * Getter for resource owner.
+     *
+     * @returns username
+     */
+    private get username(): string {
+        return this.serviceInfo.uaa.username;
+    }
+
+    /**
+     * Getter for resource owner password.
+     *
+     * @returns password
+     */
```

Comment:
Do we need doc comments for private getters? The comments are longer than the code.
I'd add that if the comments are needed to clarify the code then the getter name may need to be rethought.

Reply:
Lint requires jsdoc! Doesnt complain locally but fails remote build.